.toolversions

```diff
@@ -0,0 +1,25 @@
+cmake="
+  DeclaredVersion='3.6.0'
+  DownloadFileLinux='cmake-3.6.0-Linux-x86_64.tar.gz'
+  DownloadFileOSX='cmake-3.6.0-Darwin-x86_64.tar.gz'
+  DownloadFileWindows32='cmake-3.6.0-win32-x86.zip'
+  DownloadFileWindows64='cmake-3.6.0-win64-x64.zip'
+  DownloadUrl='https://cmake.org/files/v3.6/'
+  LocalToolFolder='Tools/downloads/cmake'
+  LocalToolFolderWindows='Tools\\downloads\\cmake'
+  LocalSearchPathLinux='cmake-3.6.0-Linux-x86_64/bin/cmake'
+  LocalSearchPathOSX='cmake-3.6.0-Darwin-x86_64/CMake.app/Contents/bin/cmake'
+  LocalSearchPathWindows32='cmake-3.6.0-win32-x86\\bin\\cmake.exe'
+  LocalSearchPathWindows64='cmake-3.6.0-win64-x64\\bin\\cmake.exe'
+  SearchPathProgramFiles32='%programfiles(x86)%\\CMake\\bin\\cmake.exe'
```

Comment:
Yes. Thanks for the tip.

---
In reply to: [111213222](https://github.com/ravimeda/corefx/pull/9#discussion_r111213222) [](ancestors = 111213222)